EIPS/eip-x.md

```diff
@@ -0,0 +1,140 @@
+| Blocks regularly under-full, user willing to wait for a good price. | User bids below the low end of the recently accepted prices, may need to wait for a while. If waiting too long, user may need to re-submit with a higher price. | User chooses their price, and waits for it, or manually re-submits.                          | User chooses their lowest price, but also their highest price and maximum wait time, so no resubmission is needed. |
+
+In all cases, the escalator algorithm as I have described is able to perform optimally.
+
+The current gas auction model works well under half-full and less conditions, but for users with urgent needs, has the downside of overpayment. For users seeking a low price, the current model has the downside of requiring re-submission, but has the benefit of always giving users a path towards reliable block inclusion.
+
+EIP-1559 also performs well under normal conditions, but under conditions where blocks are regularly full, the price discovery mechanism breaks, and miners will fall back to the `TIP` parameter to choose the transactions to include, meaning that under network congestion, EIP-1559 forces users to _either_ choose efficient prices or certainty of next-block inclusion.
+
+EIP-1559 also has all the re-submission issues of the current model in situations where a user would like to pay under the current market rate, but has certain time constraints limiting their patience. The Escalator algorithm is the only strategy listed here that allows users to discover the lowest possible price given the network conditions and their time constraints.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+**Client-Wide Parameters**
+* `INITIAL_FORK_BLKNUM`: TBD
+
+**Transaction Parameters**
+The transaction `gasPrice` parameter is now optional, and if excluded can be replaced by these parameters instead:
+
+* `START_PRICE`: The lowest price that the user would like to pay for the transaction.
+* `START_BLOCK`: The first block that this transaction is valid at.
+* `MAX_PRICE`: The maximum price the sender would be willing to pay to have this transaction processed.
+* `MAX_BLOCK`: The last block that the user is willing to wait for the transaction to be processed in.
```

Comment:
Consider changing this to `START_TIME` and `MAX_TIME` instead.  Miners can only fudge the block timestamp a little, and it is very costly for them to fudge the timestamp into the future.  Timestamps are much easier for humans, and also much more reliable (block times range from 1 second to 100 seconds) so using block numbers can result in very unexpected (for a human) wait times.

Also, by using timestamps here we will incentivize miners to more accurately set block timestamps than they do now.  Right now, many miners set timestamps far (30+ seconds) in the past because there is a *very marginal* advantage for them to have their clocks be off by a bit.  By having this be timestamp based, miners are strongly encouraged to set the timestamp as far forward as they can without risking significant profits.  It turns out, that is just about `now` because they start taking on huge risk by setting `block.timestamp` to the future.  This means that we can kill two birds with one stone with this PR by having gas price (or gas premium) be a function of time instead of block number.

Reply:
This is an interesting point, and I have no objection, so I've made this change. I hope this doesn't greatly complicate any analyses that are currently in progress.